src/packages/sdk/src/__tests__/__snapshots__/engineCommands.test.ts.snap

```diff
@@ -28492,24 +28472,6 @@ Object {
             "minNumFields": null,
           },
           "fields": Array [
-            Object {
-              "inputTypes": Array [
-                Object {
-                  "isList": false,
-                  "location": "scalar",
-                  "type": "Int",
-                },
-                Object {
-                  "isList": false,
-                  "location": "inputObjectTypes",
-                  "namespace": "prisma",
-                  "type": "IntFieldUpdateOperationsInput",
-                },
-              ],
-              "isNullable": false,
-              "isRequired": false,
-              "name": "id",
-            },
```

Comment:
What is causing this?

Reply:
We are currently investigating

Reply:
These are expected changes as this test does not have a datasource. Meaning that these operations are not supported. cc @dpetrick

Reply:
Also, the gh ui is a bit screwed up here. Probably due to the large size of the file. Basically, the snapshot indicated is incorrect. It is not `getDMMF model with autoincrement should fail if mysql` but actually `getDMMF simple model`